Engines/Wine/Plugins/virtual desktop/script.js

```diff
@@ -1,13 +1,23 @@
+// eslint-disable no-unused-vars
```

Comment:
This looks strange to me. Is it required=

Reply:
I've red the eslint documentation and it states that puting `// ealint-disable rulename` at the top disables that specific rule for this file. But Codacy still shows errors.

Reply:
You're misunderstanding the codacy issues. You're disabling the issue type `no-unused-vars`. Codacy warns you about the setting `width` and `height` without using `const`, `let` or `var` before. This issue type has the name `no-undef`. 

In my opinion you can ignore the issues, because you can't fix them currently. A possible fix for the issue is my suggestion in https://github.com/PhoenicisOrg/scripts/pull/1073#discussion_r310387402